scripts/generate_tilesets.py

```diff
@@ -392,11 +392,18 @@ def main(cfg_file_path):
         except:
             if YEAR=='multi-year':
                 logger.error("Option 'multi-year' chosen but the tile geodataframe does not contain a year column. " 
-                            "Please add it or set a year in the configuration file.")
+                            "Please add it or set a numeric year in the configuration file.")
```

Comment:
Have you tried all the options? I did some modification but I haven't retained all the option you proposed.

Reply:
These options were proposed under the FOLDER option already. I didn't test all the options, but I check mentally the conditions of the 4 options and they could all be useful. But I see what you mean, the two last options, would probably never happen as long as a year or "multi-year" is given in the config file. 

Now, that I see the code again, to avoid code duplication, should not this code be put in a function ?